import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/ImportAdapter.java

```diff
@@ -107,4 +119,36 @@ public static void main(String[] args) throws IOException, GeneralSecurityExcept
     Device device = DeviceUtil.createServerDevice(flags.dimseAET, flags.dimsePort, serviceRegistry);
     device.bindConnections();
   }
+
+  private static IDicomRedactor configureRedactor(Flags flags) throws IOException{
+    IDicomRedactor redactor = null;
+    int tagEditFlags = (flags.tagsToRemove.isEmpty() ? 0 : 1) +
+        (flags.tagsToKeep.isEmpty() ? 0 : 1) +
+        (flags.tagsProfile.isEmpty() ? 0 : 1);
+    if (tagEditFlags > 1) {
+      throw new IOException("Only one of 'tag_edit' flags may be used");
```

Comment:
IOException doesn't seem to fit this error case, maybe java.lang.IllegalArgumentException?

Reply:
Ok